package.json

```diff
@@ -114,6 +133,10 @@
     "babel-relay-plugin": "^0.9.0"
   },
   "lint-staged": {
-    "*.@(ts|tsx)": ["tslint --fix", "npm run prettier-write --", "git add"]
+    "*.@(ts|tsx)": [
+      "tslint --fix",
+      "npm run prettier-write --",
+      "git add"
+    ]
   }
 }
```

Comment:
I feel like I should [get vscode-artsy](https://github.com/artsy/vscode-artsy) up and running to ensure consistent  setups